saleor/dashboard/staff/views.py

```diff
@@ -24,8 +28,45 @@ def staff_list(request):
 def staff_details(request, pk):
     queryset = User.objects.filter(is_staff=True)
     staff_member = get_object_or_404(queryset, pk=pk)
-    form = UserGroupForm(request.POST or None, instance=staff_member)
+    form = StaffForm(request.POST or None, instance=staff_member,
+                     admin=request.user)
```

Comment:
There are some places in code where we pass `request.user` as an argument and we used to call this argument simply `user`. Here we have `staff_member` and `admin` which is a bit confusing.

Reply:
i had to differ staff_member (and his details) from user which is editing his account. And as only admin can do this i thought it would be more readable.

Reply:
That's true, but we don't use name `admin` anywhere else in code and in the context of Saleor it's not that obvious what that name would refer to. What I mean is that we should try to be consistent as much as possible.

Reply:
Ok, i will change it.